stl/inc/mutex

```diff
@@ -522,7 +521,21 @@ auto _Invoke_stored(tuple<_Types...>&& _Tuple)
     return _Invoke_stored_explicit(_STD move(_Tuple), index_sequence_for<_Types...>());
 }
 
+// STRUCT once_flag
+struct once_flag { // opaque data structure for call_once()
+    constexpr once_flag()       = default;

```

Comment:
Ditto "it seems odd to explicitly `constexpr` while letting `noexcept` be deduced".